cmd/run.go

```diff
@@ -40,6 +41,12 @@ func addRunDirFlag(cmd *cobra.Command, runDirPtr *string) {
 	cmd.Flags().StringVar(runDirPtr, "runDir", ".", "directory where experiment is run; contains experiment.yaml and result.yaml")
 }
 
+// addReuseResult allows the experiment to reuse the experiment result for
+// looping experiments
+func addReuseResult(cmd *cobra.Command, reuseResultPtr *bool) {
+	cmd.Flags().BoolVar(reuseResultPtr, "reuseResult", false, "reuse experiment result for looping experiment")
```

Comment:
I wasn't sure if we wanted a shorthand option. We currently don't have many other options for `iter8 k run` but I feel that this might be too niche to warrant it.

Reply:
agreed